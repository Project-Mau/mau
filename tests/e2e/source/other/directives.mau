= Test directives

This file tests Mau directives

::#include:tests/e2e/source/other/include_text.txt
