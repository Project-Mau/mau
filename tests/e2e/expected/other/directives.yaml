data:
  args: []
  content:
  - data:
      anchor: test-directives-93d4
      args: []
      kwargs: {}
      level: 1
      subtype: null
      tags: []
      type: header
      value:
        data:
          args: []
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: Test directives
          kwargs: {}
          subtype: null
          tags: []
          type: sentence
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: This file tests Mau directives
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: This text has been included through a directive
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  kwargs: {}
  subtype: null
  tags: []
  type: document

